CondTools/SiPhase2Tracker/plugins/DTCCablingMapTestProducer.cc

```diff
@@ -0,0 +1,110 @@
+// -*- C++ -*-
+//
+// Package:    CondTools/SiPhase2Tracker
+// Class:      DTCCablingMapTestProducer
+// 
+/**\class DTCCablingMapTestProducer DTCCablingMapTestProducer.cc CondTools/SiPhase2Tracker/plugins/DTCCablingMapTestProducer.cc
+
+Description: [one line class summary]
+
+Implementation:
+		[Notes on implementation]
+*/
+//
+// Original Author:  Luigi Calligaris, SPRACE, São Paulo, BR
+// Created        :  Wed, 27 Feb 2019 21:41:13 GMT
+//
+//
+
+#include <memory>
+
+#include <unordered_map>
+#include <utility>
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+
+#include "CondFormats/Common/interface/Time.h"
+#include "CondFormats/SiPhase2TrackerObjects/interface/TrackerDetToDTCELinkCablingMap.h"
+#include "CondFormats/SiPhase2TrackerObjects/interface/DTCELinkId.h"
+#include "CondFormats/DataRecord/interface/TrackerDetToDTCELinkCablingMapRcd.h"
+
+
+class DTCCablingMapTestProducer : public edm::one::EDAnalyzer<>
+{
+	public:
+		explicit DTCCablingMapTestProducer(const edm::ParameterSet&);
+		~DTCCablingMapTestProducer() override;
+		
+		static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+		
+	private:
+		void beginJob() override;
+		void analyze(const edm::Event&, const edm::EventSetup&) override;
+		void endJob() override;
+private:
+		cond::Time_t                                    iovBeginTime_;
+		std::unique_ptr<TrackerDetToDTCELinkCablingMap> pCablingMap_ ;
+		std::string                                     recordName_  ;
+};
+
+void DTCCablingMapTestProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions)
+{
+	edm::ParameterSetDescription desc;
+	desc.setComment("Stores a dummy TrackerDetToDTCELinkCablingMap database object from a CSV file.");
+  desc.add<long long unsigned int>("iovBeginTime", 1);
+  desc.add<std::string>("record","TrackerDetToDTCELinkCablingMap");
```

Comment:
Please also patch the inconsistent indent here.

Reply:
Ok, fixed!